misc/windows-deploy/hostsetup.ps1

```diff
@@ -11,41 +11,119 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
+<#
+.NOTES
+This setup script is required in order to enable task IAM roles on windows instances.
+
+.NOTES
+169.254.170.2:51679 is the IP address used for task IAM roles.
```

Comment:
169.254.170.2:80 is the correct IP+port.  There's port translation that happens to convert 80 to 51679, which is where the agent is actually listening.

Reply:
Is your comment about the code comment, or about the code behavior? Could you clarify the proposed change?

The behavior wasn't changed in this PR for the port. See the current file:
https://github.com/aws/amazon-ecs-agent/blob/master/misc/windows-deploy/hostsetup.ps1

# 169.254.170.2:51679 is the IP address used for task IAM roles.
$credentialAddress = "169.254.170.2"
$credentialPort = "51679"

Reply:
sry for the weird text... im getting used to github comment syntax.

Reply:
ECS Agent listens on port `51679` for credentials, which is basically `127.0.0.1:51679`. `169.254.170.2:80` is redirected to `127.0.0.1:51679` so that containers don't need to worry/be aware of that.